### PR TITLE
Updated OSCThreadServer to always use select.

### PR DESCRIPTION
**Pull request by CJRChang**
Fix for https://github.com/kivy/oscpy/issues/6 to avoid server timeout when only one socket is used.

### CODE REVIEW

File: oscpy/server.py [hunk 1]
```diff
@@ -327,7 +325,7 @@ def send_bundle(
         send_bundle(messages, ip_address, port, sock=sock, safer=safer)
 
     def answer(
-        self, address=None, values=None, bundle=None, timetag=None, safer=False
+        self, address=None, values=None, bundle=None, timetag=None, safer=False, port=None
```
Comment (tshirtman) [inline oscpy/server.py]: please split line after to stick to pep8 (before "safer", because the comma hit the 80 mark).

File: oscpy/server.py [hunk 2]
```diff
@@ -350,16 +348,18 @@ def answer(
         else:
             raise RuntimeError('answer() not called from a callback')
 
-        ip_address, port = frame.f_locals.get('sender')
+        ip_address, responseport = frame.f_locals.get('sender')
```
Comment (tshirtman) [inline oscpy/server.py]: compound names use an underscore, so please make it `response_port`.

File: oscpy/server.py (hunk 1 above)
Comment (CJRChang) [reply oscpy/server.py]: Urgh yes - sorry, this was done in haste. Will update.

Comment (tshirtman) [reply oscpy/server.py]: No worries, i'm seeing i was far from perfect on the pep8 side, fixing the other ones on my side :)

**Comment by CJRChang** [comment]
Thanks for the guidance, updated as requested.
I'd write tests but I'm stuck on a Windows machine at the moment, I'll find time to circle back when I'm on another machine.

**Pull request merged**